MailCore/Utils/SnackBar/IKSnackBar+Extension.swift

```diff
@@ -40,7 +41,8 @@ public extension SnackBarStyle {
     }
 }
 
-public class SnackBarAvoider {
+// TODO: delete
+public final class SnackBarAvoider {
```

Comment:
Question: By what is it replaced ? Maybe I missed something ?

Reply:
Good catch, I needed to remove it.

It was renamed `IKSnackBarAvoider` in CoreUI to match core UI termminology.